src/test/java/seedu/address/logic/commands/EditPersonDescriptorTest.java

```diff
@@ -65,8 +75,10 @@ public void toStringMethod() {
                 + editPersonDescriptor.getPhone().orElse(null) + ", email="
                 + editPersonDescriptor.getEmail().orElse(null) + ", address="
                 + editPersonDescriptor.getAddress().orElse(null) + ", priority="
-                + editPersonDescriptor.getPriority().orElse(null) + ", remark="
-                + editPersonDescriptor.getRemark().orElse(null) + ", tags="
+                + editPersonDescriptor.getIncome().orElse(null) + ", remark="
+                + editPersonDescriptor.getRemark().orElse(null) + ", dateOfBirth="
+                + editPersonDescriptor.getDateOfBirth().orElse(null) + ", income="
```

Comment:
Seems like the income and priority strings got swapped when adding in the new fields?

Reply:
Noted, the ordering will be corrected in the next commit